docs/en/integrations/language-clients/java/index.md

```diff
@@ -155,6 +165,17 @@ Consider [clickhouse-client](/docs/en/integrations/clickhouse-client-local.md) w
     <classifier>all</classifier>
 </dependency>
 ```
+
+Since version `0.5.0` we are using Apache HTTP Client we are packing the Client, but since there is not a sharded version we need to have a logger as dependency (if not already using please include as dependency).
```

Comment:
```suggestion
Since version `0.5.0`, we are using Apache HTTP Client that's packed the Client. Since there is not a shared version of the package, you need to add a logger as a dependency.
```